CHANGELOG.md

```diff
@@ -19,13 +22,20 @@ ___Note:__ Yet to be released changes appear here._
 * `FEAT`: validate receive task after event-based gateway ([#3569](https://github.com/camunda/camunda-modeler/issues/3569))
 * `FEAT`: support signal throw and end event in Camunda 8.3 ([#3555](https://github.com/camunda/camunda-modeler/issues/3555))
 * `FEAT`: variable suggestions take the expression position into account ([\`@bpmn-io/variable-resolver#19](https://github.com/bpmn-io/variable-resolver/pull/19), [#3510](https://github.com/camunda/camunda-modeler/issues/3510))
+* `FEAT`: remove templated `bpmn:Message` if no message bindings are active ([bpmn-js-properties-panel#915](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/915))
+* `FEAT`: allow time date in boundary and intermediate catch events ([bpmn-js-properties-panel#931](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/931))
+FEAT: allow time date for timer intermediate catch and boundary event in Camunda 8.3 (98)
 * `FIX`: handle missing `resultExpression` value ([\`@bpmn-io/variable-resolver#20](https://github.com/bpmn-io/variable-resolver/pull/20), [#3599](https://github.com/camunda/camunda-modeler/issues/3599))
```

Comment:
Just found this one, it was broken before this PR but let's fix the typo while we're at it
```suggestion
* `FIX`: handle missing `resultExpression` value ([@bpmn-io/variable-resolver#20](https://github.com/bpmn-io/variable-resolver/pull/20), [#3599](https://github.com/camunda/camunda-modeler/issues/3599))
```